manifests/t/tawk/to/2.2.0/tawk.to.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: tawk.to
+PackageVersion: 2.2.0
+PackageLocale: en-US
+Publisher: tawk.to, inc.
+PackageName: tawk.to Windows App
+License: tawk.to Terms of Services
+Copyright: Copyright © 2020 tawk.to
+ShortDescription: Windows App to monitor and chat with the visitors on your website, respond to support tickets, organize contacts and create a help center to empower customers to help themselves.

```

Comment:
```suggestion
Publisher: tawk.to
PublisherUrl: https://www.tawk.to/
PublisherSupportUrl: https://help.tawk.to/
PrivacyUrl: https://www.tawk.to/privacy-policy/
# Author:
PackageName: tawk.to
PackageUrl: https://www.tawk.to/tawk-to-for-windows/
License: Proprietary
LicenseUrl: https://www.tawk.to/terms-of-service/
Copyright: Copyright © 2020 tawk.to
# CopyrightUrl:
ShortDescription: Windows App to monitor and chat with the visitors on your website, respond to support tickets, organize contacts and create a help center to empower customers to help themselves.
# Description:
Moniker: tawk
```
Making these all as one suggestion to make it easy to commit -
* Publisher should match the value written to Control Panel
* PackageName should match the value written to Control Panel
* Added additional metadata